src/app/caesar-portal/group-list/group-list.component.html

```diff
@@ -1,31 +1,29 @@
 <div class="col-md-3 group-list-container">
   <div class="row">
-    <div class="col-md-6">
-      <i class="glyphicon glyphicon-search"></i>
-    </div>
-    <div class="col-md-6" *ngIf="groupsQuantity > itemsPerPage">
+    <div class="col-md-12" *ngIf="groupsQuantity > itemsPerPage">
       <caesar-pager [currentPage]="currentPage" [itemsQuantity]="groupsQuantity" [itemsPerPage]="itemsPerPage" (pageChange)="onPageChange($event)"></caesar-pager>
     </div>
   </div>
   <div class="row">
-  <div class="col-md-6">
-    <a href='#'><span class='glyphicon glyphicon-plus search-btn'></span></a>
-  </div>
-  <div class="col-md-6">
+    <div class="col-md-12">
+      <a href='#'><span class='glyphicon glyphicon-plus search-btn'></span></a>
+    </div>
   </div>
-</div>
   <div class="row">
-    <div class="col-md-5 " *ngFor="let item of (groups | limitTo : firstItem : lastItem)">
-      <group-item [group]="item">Group</group-item>
+    <div class="col-md-6" *ngFor="let item of (groups | limitTo : firstItem : lastItem | mathcesProgress : groupStatus )">
```

Comment:
change this line according to
<div class="col-md-6" *ngFor="let item of (groups | mathcesProgress : groupStatus | limitTo : firstItem : lastItem )">